src/pal/test/snp_ioctl_test.cpp

```diff
@@ -82,6 +82,13 @@ TEST_CASE("SNP derived keys with different TCBs should be different")
 
 int main(int argc, char** argv)
 {
+  if (!ccf::pal::snp::supports_sev_snp())
+  {
+    std::cout << "Skipping all tests as this is not running in SEV-SNP"
+              << std::endl;
+    return 0;
+  }
```

Comment:
Previously disabled at build-time somewhere in CMake. Now built and run all the time, and a run-time capability probe determines whether it's actually testing anything.